blockchain/beacon/src/drand.rs

```diff
@@ -59,6 +60,9 @@ pub struct DrandBeacon {
     drand_gen_time: u64,
     fil_gen_time: u64,
     fil_round_time: u64,
+
+    // Keeps track of computed beacon entries.
+    local_cache: HashMap<u64, BeaconEntry>,
```

Comment:
traditional hashmaps are quite slow for integer values, do you mind switching this to [ahash](https://docs.rs/ahash/0.4.4/ahash/index.html) or [fnv hash map](https://docs.rs/fnv/1.0.7/fnv/) (I would say ahash because it's slightly quicker but maybe the argument could be made for fnv)

Reply:
sure, updated to `ahash`.